packages/story-editor/src/components/form/tags/input.js

```diff
@@ -135,48 +171,110 @@ function Input({
+        } else {
+          menuRef?.current?.children?.[previousChild]?.focus();
+        }
+      }
+      if (e.key === 'Enter') {
+        handleTagSelectedFromSuggestions(e, name);
+      }
+    },
+    [handleTagSelectedFromSuggestions, totalSuggestions]
+  );
+
   return (
-    <Border isInputFocused={isInputFocused}>
-      {
-        // Text input should move, relative to end, with offset
-        // this helps with natural tab order and visuals
-        // as you ArrowLeft or ArrowRight through tags
-        [
-          ...renderedTags.slice(0, renderedTags.length - offset),
-          INPUT_KEY,
-          ...renderedTags.slice(renderedTags.length - offset),
-        ].map((tag) =>
-          tag === INPUT_KEY ? (
-            <TextInput
-              {...props}
-              key={INPUT_KEY}
-              value={value}
-              onKeyDown={handleKeyDown}
-              onChange={handleChange}
-              onFocus={handleFocus}
-              onBlur={handleBlur}
-              size="4"
-            />
-          ) : (
-            <Tag key={tag} onDismiss={removeTag(tag)}>
-              {tagDisplayTransformer(tag) || tag}
-            </Tag>
+    <Border ref={containerRef} isInputFocused={isInputFocused}>
+      <InputWrapper>
+        {
+          // Text input should move, relative to end, with offset
+          // this helps with natural tab order and visuals
+          // as you ArrowLeft or ArrowRight through tags
+          [
+            ...renderedTags.slice(0, renderedTags.length - offset),
+            INPUT_KEY,
+            ...renderedTags.slice(renderedTags.length - offset),
+          ].map((tag) =>
+            tag === INPUT_KEY ? (
+              <TextInput
+                {...props}
+                key={INPUT_KEY}
+                value={value}
+                onKeyDown={handleKeyDown}
+                onChange={handleChange}
+                onFocus={handleFocus}
+                onBlur={handleBlur}
+                size="4"
+                ref={inputRef}
+                autoComplete="off"
+              />
+            ) : (
+              <Tag key={tag} onDismiss={removeTag(tag)}>
+                {tagDisplayTransformer(tag) || tag}
+              </Tag>
+            )
           )
-        )
-      }
+        }
+      </InputWrapper>
+      {suggestedTerms.length > 0 && (
+        <SuggestionList
+          aria-label={suggestedTermsLabel}
+          role="listbox"
+          ref={menuRef}
+          data-testid="suggested_terms_list"
+        >
+          {suggestedTerms.map(({ name, id }, index) => (
+            <li
+              key={id}
+              aria-selected={value === name}
+              role="option"
+              tabIndex={0}
+              onClick={(e) => handleTagSelectedFromSuggestions(e, name)}
+              onKeyDown={(e) => handleSuggestionKeyDown(e, index, name)}
+            >
+              {name}
+            </li>
+          ))}
+        </SuggestionList>
+      )}
```

Comment:
💭 For accessibility: when this appears I think it would be helpful to broadcast a message about how many 'results' there are. We can use the `useLiveRegion` hook for this

Reply:
let's hold on adding a live region here? It feels like it might be disruptive?

Reply:
hmm I feel like it's okay if it's disruptive, but let's at least move this to another ticket 😄 It may not be needed with the `autocomplete="on"` 🤔